EMIP/1-1000/EMIP00322.xml

```diff
@@ -29,17 +30,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </availability>
          </publicationStmt>
          <sourceDesc>
-            <msDesc xml:id="ms">
+             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0447EMIP">Weiner</repository>
                   <collection>EMIP</collection>
                   <idno facs="EMIP/Codices/322/">Weiner Codex 132</idno>
                </msIdentifier>
                <msContents>
                   <summary>Canticle of the Flower, ማሕሌተ፡ ጽጌ፡, Praises of Mary, ውዳሴ፡ ማርያም፡, and other prayers</summary>
-                  <msItem xml:id="ms_i1">
                      <textLang mainLang="gez">Gə‘əz</textLang>
-                  </msItem>
+                     <msItem xml:id="ms_i1"><locus from="1r" to="44v"/><title type="complete" ref="LIT1834Mahlet"/></msItem>
```

Comment:
could you please add also `<textLang mainLang="gez" />` as appropriate at least in the upper level `msItems`?